pkg/controller/config/config_controller.go

```diff
@@ -91,13 +96,16 @@ func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 		community = mf.Manifest{}
 	}
 
+	kc, _ := k8s.NewForConfig(mgr.GetConfig())
+
 	return &ReconcileConfig{
 		client:    mgr.GetClient(),
 		scheme:    mgr.GetScheme(),
 		pipeline:  pipeline,
 		triggers:  triggers,
 		addons:    addons,
 		community: community,
+		kc:        kc,
```

Comment:
are the type clients necessary.
could we use the dynamic client as it is already in `ReconcileConfig`.

Reply:
The dynamic client wasn't working 🤔 . Wondering that's why the dynamic client wasn't used in the first place in the other controller 🤔 I can give it another try if you tell me :)